styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -290,6 +290,59 @@
   text-align: right;
 }
 
+//Case Study
+  
+.caseStudy-wrapper {
+  background-image: url('../imgs/derek/banners/SSImage.jpg');
```

Comment:
![image](https://user-images.githubusercontent.com/5263371/44418282-94a17f00-a53d-11e8-863e-0b635bbd5413.png)

Do we care about how the kid's face gets chopped in wider browsers?

Reply:
@WidgetsBurritos That's a Mike question again. The image will change depending on the client we feature. To make it work for whatever client I'll need to edit the images individually. So if I adjust where this image sits, the next client image could look terrible. It's just like the banners. It's part of why full image backgrounds arent my favorite thing.